src/main/java/com/ncc/aif/ValidateAIF.java

```diff
@@ -41,20 +41,23 @@
     private static final String INTERCHANGE_URI = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/InterchangeOntology";
     private static final String AIDA_DOMAIN_COMMON_URI = "https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/AidaDomainOntologiesCommon";
 
-    private Model domainModel;
     private static Model shaclModel;
     private static Model nistModel;
+    static {
+        shaclModel = ModelFactory.createOntologyModel();
+        CharSource aifSource = Resources.asCharSource(Resources.getResource(AIDA_SHACL_RESNAME), Charsets.UTF_8);
+        loadModel(shaclModel, aifSource);
+
+        nistModel = ModelFactory.createOntologyModel();
+        loadModel(nistModel, aifSource);
+        loadModel(nistModel, Resources.asCharSource(Resources.getResource(NIST_SHACL_RESNAME), Charsets.UTF_8));
+    }
```

Comment:
Good call. I'd like to avoid having the class rely on a particular logger, but I will move the static initialization to be called the first time a ValidateAIF instance is created.

Reply:
Ah yes, that's a better solution-- thanks.  Programmatic use of the validator shouldn't involve any particular logger.